tools/cleanup_mri_tables_for_19-0_release.php

```diff
@@ -263,6 +263,11 @@ function backupEntries($selectID, $table_name, $FK_field)
     $IDs = generateIdList($selectID, $FK_field);
     $where = $FK_field . " IN (" . $IDs . ")";
 
+    // create directory where the back up will go if it does not exist yet
+    if ( !file_exists("../project/backup") ) {
```

Comment:
these should include the `__DIR__` see line 273

otherwise someone running it from another directory than tools would have issues